frontend/src/scenes/data-warehouse/editor/multitabEditorLogic.tsx

```diff
@@ -853,6 +853,8 @@ export const multitabEditorLogic = kea<multitabEditorLogicType>([
                 view.query?.query !== latestView?.query.query
             ) {
                 actions._setSuggestionPayload({
+                    // undefined satisfies types, queryInput shouldn't be null here
+                    suggestedValue: values.queryInput || undefined,
```

Comment:
can you use `values.queryInput!` to tell type checking we know it's not null? don't know if we know for sure that it's not null or it's just shouldn't be. non-blocking